package.json

```diff
@@ -34,12 +34,17 @@
   "author": "HomeOffice",
   "dependencies": {
     "browserify": "^12.0.1",
-    "hof-bootstrap": "^6.0.0",
-    "hof-frontend-toolkit": "^1.0.0",
+    "express": "^4.14.0",
```

Comment:
There is nothing wrong with the linting. It (correctly) assumes everything in `./test` is a dev concern and everything else is a prod concern.

In this case, there is an exception, so you need to override it with manual config.